### PR TITLE
New package: dnscrypt-proxy-1.6.0

### PR DESCRIPTION
**Pull request by ybkbolio**

### CODE REVIEW

File: srcpkgs/dnscrypt-proxy/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'dnscrypt-proxy'
+pkgname=dnscrypt-proxy
+version="1.6.0"
+revision=1
+build_style=gnu-configure
+makedepends="libsodium-devel"
+depends="libsodium"
```
Comment (xtraeme) [inline srcpkgs/dnscrypt-proxy/template]: This depends line is wrong.

File: srcpkgs/dnscrypt-proxy/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="DNS proxy that encrypts queries"
+maintainer="kebolio <keboler@cock.li>"
+license="ISC"
+homepage="https://dnscrypt.org/"
+distfiles="https://download.dnscrypt.org/dnscrypt-proxy/dnscrypt-proxy-1.6.0.tar.gz"
```
Comment (xtraeme) [inline srcpkgs/dnscrypt-proxy/template]: Use `${pkgname}` and `${version}`

File: srcpkgs/dnscrypt-proxy/template (hunk 1 above)
Comment (ybkbolio) [inline srcpkgs/dnscrypt-proxy/template]: On 28 December 2015 08:10:55 GMT+00:00, Juan RP <notifications@github.com> wrote:
>> @@ -0,0 +1,20 @@
>> +# Template file for 'dnscrypt-proxy'
>> +pkgname=dnscrypt-proxy
>> +version="1.6.0"
>> +revision=1
>> +build_style=gnu-configure
>> +makedepends="libsodium-devel"
>> +depends="libsodium"
>
>This depends line is wrong.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/voidlinux/void-packages/pull/3238/files#r48467014

does libsodium need a version bit tacked onto it then?

Comment (xtraeme) [inline srcpkgs/dnscrypt-proxy/template]: the runtime dependency version is set by `common/shlibs` and it's set automagically by xbps-src.

Comment (ybkbolio) [inline srcpkgs/dnscrypt-proxy/template]: On 28 December 2015 08:37:12 GMT+00:00, Juan RP <notifications@github.com> wrote:
>> @@ -0,0 +1,20 @@
>> +# Template file for 'dnscrypt-proxy'
>> +pkgname=dnscrypt-proxy
>> +version="1.6.0"
>> +revision=1
>> +build_style=gnu-configure
>> +makedepends="libsodium-devel"
>> +depends="libsodium"
>
>the runtime dependency version is set by `common/shlibs` and it's set
>automagically by xbps-src.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/voidlinux/void-packages/pull/3238/files#r48467813

so the depends line is redundant?

also i realised that dnscrypt-proxy runs logs to stderr, so i will add a log file when i get back

**Pull request merged**